commands/random/coinflip.js

```diff
@@ -0,0 +1,22 @@
+const {
+  SlashCommandBuilder,
+  EmbedBuilder,
+  AttachmentBuilder,
+} = require("discord.js");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("coinflip")
+    .setDescription("Flips a coin!"),
+  async execute(interaction) {
+    const randomNum = Math.round(Math.random());
+    const headsOrTails = randomNum == 0 ? "Heads!" : "Tails!";
+    const fileName = randomNum == 0 ? "Heads.png" : "Tails.png";
```

Comment:
Instead of having file names, put the image url here. Less storage is being used for the same speed of image loading.

Reply:
Basically don't have local files. Right click the image on the web and say copy image address.